example/lambdas/python-reference-task/package.json

```diff
@@ -14,7 +14,7 @@
     "python-lint": "pipenv run pylint *.py",
     "lint": "npm run python-lint",
     "clean": "rm -rf dist && rm -rf lib && rm -rf .venv && mkdir dist && mkdir lib",
-    "build": "pip install pipenv && PIPENV_VENV_IN_PROJECT=1 pipenv install --dev --deploy --ignore-pipfile",
+    "build": "pip install pipenv && PIPENV_VENV_IN_PROJECT=1 PIPENV_NOSPIN=1 pipenv install -vvv --dev --deploy --ignore-pipfile",
```

Comment:
Do we intend to leave the `-vvv` in the example or was that just for debugging? Fine either way, just want to make sure it's intentional.

Reply:
Nope, it's just the type of thing I'd accidentally leave in until the review 😄